locale/action_names.rb

```diff
@@ -1,73 +1,75 @@
-_("Update")
-_("Remove Content")
+_("Create")
+_("Delete")
+_("Delete Package Group")
+_("Discover")
+_("Errata mail")
+_("Export")
+_("Fetch pxe files")
+_("Filtered index content")
 _("Upload into")
+_("Index content")
+_("Index errata")
 _("Index module streams")
+_("Index package groups")
 _("Instance update")
-_("Remove from Environment")
-_("Remove Versions and Associations")
+_("Remove Content")
 _("Synchronize")
-_("Publish")
+_("Update")
 _("Update content urls")
-_("Promote")
+_("Update http proxy details")
+_("Update redhat repository")
+_("Upload errata into")
 _("Create Package Group")
+_("Verify checksum")
 _("Disable")
-_("Errata mail")
-_("Generate host applicability")
-_("Agent action")
-_("Delete Activation Key")
-_("Create")
-_("Abstract async task")
-_("Remote action:")
+_("Enable")
+_("Run Sync Plan:")
+_("Create Export History")
+_("Create Import History")
+_("Copy version units to library")
 _("Export Library")
 _("Import")
+_("Remote action:")
 _("Import Puppet classes")
 _("Import facts")
-_("Index package groups")
-_("Index errata")
-_("Index content")
 _("Action with sub plans")
+_("Abstract async task")
+_("Delete Activation Key")
+_("Agent action")
+_("Generate host applicability")
 _("Bulk generate applicability for hosts")
 _("Generate repository applicability")
 _("Synchronize smart proxy")
 _("Sync capsule")
-_("Delete")
 _("Incremental Update of  Content View Version(s) ")
+_("Promote")
 _("Promotion to Environment")
+_("Publish")
+_("Remove from Environment")
 _("Remove Version")
-_("Filtered index content")
-_("Fetch pxe files")
-_("Export")
-_("Discover")
-_("Delete Package Group")
-_("Update http proxy")
-_("Reindex subscriptions")
-_("Delete Product")
-_("Product Create")
-_("Destroy")
-_("Combined Profile Update")
-_("Package Profile Update")
-_("Updating System Purpose for host")
-_("Update release version for host")
-_("Update for host")
-_("Update Content Overrides")
-_("Remove subscriptions")
-_("Hypervisors update")
-_("Hypervisors")
-_("Install Applicable Errata")
-_("Destroy Content Host")
-_("Auto attach subscriptions")
-_("Attach subscriptions")
+_("Remove Versions and Associations")
+_("Import Content View Version")
+_("Import Default Content View")
+_("Incremental Update")
 _("Publish Lifecycle Environment Repositories")
+_("Attach subscriptions")
+_("Auto attach subscriptions")
+_("Destroy Content Host")
+_("Install Applicable Errata")
+_("Hypervisors")
+_("Hypervisors update")
+_("Remove subscriptions")
+_("Update Content Overrides")
+_("Update for host")
+_("Update release version for host")
+_("Updating System Purpose for host")
+_("Package Profile Update")
+_("Combined Profile Update")
+_("Destroy")
+_("Product Create")
+_("Delete Product")
+_("Reindex subscriptions")
+_("Update http proxy")
 _("Delete Lifecycle Environment")
-_("Republish Version Repositories")
-_("Incremental Update")
-_("Import Default Content View")
-_("Import Content View Version")
+_("Republish Version Repositories")
```

Comment:
You'll need an `echo "" >> ~/katello/locale/action_names.rb` here to stop RuboCop from complaining.